### ISSUE TITLE
De-unicodify internal API

### ISSUE THREAD

**Issue by tkelman** [JuliaLang/JuliaParser.jl#36]
This package isn't doing math, so `√` isn't appropriate. The meaning on `¬` is pretty unconventional too, and `⨳, ⪥, ⤄` are tough to read. Would have been nice to open `kf/loctrack` as a PR for feedback before merging it into master.

**Comment by jakebolewski** [comment]
I tend to agree, it is nice that it is terse for readability but at the same time it is much harder to edit and the non-combiner symbols are not intuitive at all.

**Comment by Ismael-VC** [comment]
It would be nice to document all the functions including this ones, what ever identifier name they end up having. I'd like to improve overall documentation and add more examples.

**Comment by jakebolewski** [comment]
@Keno this was a great improvement, but to be honest the unicode symbols make me a bit sad. Have you thought about ASCII equivalents to these function names?  It seems best to avoid code churn going forward if this is going to be part of the core infrastructure.
